File: app/src/main/java/com/mmdev/meetups/ui/adapters/ClickChatAttachmentsFirebase.java
package com.mmdev.meetups.ui.adapters;

import android.view.View;

public interface ClickChatAttachmentsFirebase
{

	/**
	 * click attached photo in chat
	 * @param view your view
	 * @param position pos
	 * @param nameUser sender name
	 * @param urlPhotoUser photo profile url sender
	 * @param urlPhotoClick clicked photo in chat url
	 */
	void clickImageChat(View view, int position, String nameUser, String urlPhotoUser, String urlPhotoClick);

	/**
	 * click attached geoposition in chat
	 * @param view your view
	 * @param position pos
	 * @param latitude latitude
	 * @param longitude longitude
	 */
	void clickMapChat (View view, int position, String latitude, String longitude);

}

File: app/src/main/java/com/mmdev/meetapp/ui/chat/ClickChatAttachmentsFirebase.kt
package com.mmdev.meetapp.ui.chat

import android.view.View

interface ClickChatAttachmentsFirebase {

	/**
	 * click attached photo in chat
	 * @param view your view
	 * @param position pos
	 * @param nameUser sender name
	 * @param urlPhotoUser photo profile url sender
	 * @param urlPhotoClick clicked photo in chat url
	 */
	fun clickImageChat(view: View, position: Int, nameUser: String, urlPhotoUser: String, urlPhotoClick: String)

	/**
	 * click attached geoposition in chat
	 * @param view your view
	 * @param position pos
	 * @param latitude latitude
	 * @param longitude longitude
	 */
	fun clickMapChat(view: View, position: Int, latitude: String, longitude: String)

}
